qipr_approver/approver/forms/about_you_form.py

```diff
@@ -61,3 +61,17 @@ def __init__(self, user=User()):
                             'options': [item.name for item in Suffix.objects.all()],
                             'selected': [item.name for item in user.person.suffix.all()]}
 
+        self.last_login = {'name': 'last_login',
+                           'label': 'Last Login',
+                           'type': 'text',
+                           'value': user.person.last_login_time or ''}
```

Comment:
They actually look fine - Aug. 17, 2016, 9:27 p.m.  However, upon seeing that type, it makes sense to make it use a locale based datetime.